entry.go

```diff
@@ -108,16 +113,13 @@ func (te *TunaEntry) Start() {
 			if err != nil {
 				return
 			}
-			stream, err := session.OpenStream()
-			if err != nil {
-				return
-			}
-			stream.Close()
+
```

Comment:
This is still not fixed: you should NOT remove this part. This is needed to keep TCP open if no traffic is going through tuna in the forward mode.